test/e2e/scalability/density.go

```diff
@@ -673,7 +709,10 @@ var _ = SIGDescribe("Density", func() {
 					time.Sleep(200 * time.Millisecond)
 				}
 				wg.Wait()
+				latencyPodStartupPhase.End()
 
+				latencyMeasurementPhase := testPhaseDurations.StartPhase(810, "pod startup latency measurements")
```

Comment:
'measuring pod latencies' seems more crisp now that I think of it.

Reply:
Done.

Reply:
Actually "pod latencies measurement" might be better. Naming.. sigh :|

Reply:
Done.